docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -69,12 +69,14 @@ Formally, an **action** is one of the following:
    The action also contains **actors**, the parties who fetch the contract.
    A **Fetch** behaves like a non-consuming exercise with no consequences, and can be repeated.
 
+#. a **Key assertion** records the assertion that the given :ref:`contract key <daml-ref-contract-keys>` is not assigned to any active contract on the ledger.
+
 An **Exercise** or a **Fetch** action on a contract is said to **use** the contract.
 Moreover, a consuming **Exercise** is said to **consume** (or **archive**) its contract.
 
 The following EBNF-like grammar summarizes the structure of actions
 and transactions. Here, "s | t" represents the choice between `s` and
-`t`, "s t" represents `s` followed by `t`, and "s*" represents the
+`t`, "s t" represents `s` followed by `t` and "s*" represents the
```

Comment:
I think the serial comma was useful here, as the list items are long and some of them contain "and", so it helps disambiguate.

Reply:
You're right. I inadvertently deleted the comma.